### ISSUE TITLE
Open search on boot

### ISSUE THREAD

**Issue by SUPERCILEX** [pop-os/gnome-shell-extension-pop-cosmic#333]
Gnome opens the workspace view on boot which allows you to search for an app right away. There should be an option to open the cosmic search on boot too.

**Comment by jackpot51** [comment]
Pop allows the use of desktop icons and a dock, which can make such a thing an inconvenience for some users.

**Comment by SUPERCILEX** [comment]
Yeah, I'm not advocating for a default, just an option for those that want to save a click.
